plugins/woocommerce/src/Internal/Orders/OrderAttributionController.php

```diff
@@ -395,7 +410,7 @@ private function log( string $message, string $method, string $level = WC_Log_Le
 	 */
 	private function send_order_tracks( array $source_data, WC_Order $order ) {
 		$origin_label        = $this->get_origin_label(
-			$source_data['type'] ?? '',
+			$source_data['source_type'] ?? '',
```

Comment:
I think this might have snuck in from the previous PR
```suggestion
			$source_data['type'] ?? '',
```

Reply:
This one will actually affect the Tracks reporting since `$source_data` has `type` but no `source_type`. So the `origin` label always ends up being `unknown`

Reply:
Reverted in 4992aaeb65d3b2552bdd7cf4f3979aa429625eeb